config/routes.rb

```diff
@@ -92,7 +92,10 @@
       get "reports/bulk" => "reports#bulk"
       get "reports/:type" => "reports#show"
 
-      resources :groups, constraints: AdminConstraint.new do
+      resources :groups, only: [:create]
+      resources :groups, except: [:create], constraints: AdminConstraint.new do
```

Comment:
Is this a sensible way to move create out of the AdminConstraint (and in to the StaffConstraint on line 78)?

Reply:
I think this is the correct way to do it, because it's no longer restricted to Admins, and the create action handles the guardian check.